src/main/java/com/mael/ttt/Marks.java

```diff
@@ -0,0 +1,18 @@
+package com.mael.ttt;
+
+public enum Marks {
+
+    PLAYER {
+        @Override
+        public String toString() {
+            return "X";
+        }
```

Comment:
You can add custom methods to an enum, which is nicer than overriding the toString() method. 
By overriding the toString, its not clear that X or O will be returned.

You could consider do something like this:

`public class enum {
PLAYER("X"), OPPONENT("O")

String symbol;
Marks(String symbol) {
this.symbol = symbol;
    String getSymbol() {
}

public void getSymbol() {
     return symbol;
}
}`

We can pair on this tomo if you like.